docs/source/install/index.rst

```diff
@@ -9,6 +9,12 @@ single system, or you can follow the manual instructions for your OS.
 
 Here's an overview of the options:
 
+* **Vagrant / Virtual Appliance:** Vagrant / OVA is a quick and easy way to try StackStorm.
+  It's already pre-installed, tested and shipped as a virtual image and so saves your time going
+  through time-consuming installation and configuration steps. Works best as a testing,
+  pack development or demo system and recommended to get familiar with StackStorm platform.
+  ``vagrant init stackstorm/st2 && vagrant up`` is all you need to get started.
+  See :doc:`Vagrant </install/vagrant>` for more detailed instructions.
 * **One-line Install:** Run our installation script for an opinionated install of all components
```

Comment:
I'm thinking about moving the `one-line-install` into separated doc and highlight `vagrant up` way in `index.rst`?

This way we'll push the pre-installed and pre-configured Vagrant box as a way to try & evaluate StackStorm.
This can save some resources in support where people are running `curl|bash` and have issues.

Thoughts?

Reply:
Or alternative, more soft way is to add a new block `Try & Eval` (or similar) before the
https://docs.stackstorm.com/install/index.html#ref-one-line-install `curl|bash` paragraph.

Reply:
You can bump up the priority, but I don't want to have Vagrant over-shadowing everything else.